### DISCUSSION TITLE
new rms voice testing

### DISCUSSION THREAD

**Discussion by zstanecic** [RHVoice/RHVoice#512] [Show and tell]
Hi!
This is the experiment version of the RMS voice. Who can tel me, is it a really good thing? Is it worth it? Rename .zip to nvda-addon to test it with NVDA>
[RHVoice-voice-English-Rms.zip](https://github.com/RHVoice/RHVoice/files/8311967/RHVoice-voice-English-Rms.zip)

**Comment by king-dahmanus** [comment]
Yeah, I think it's a really good voice. The only thing it needs is to be
converted to 22khz just like how slt and liubov sound. Speaking of
experiments, how long did it take for you to build the voice?

**Comment by zstanecic** [comment]
Hi,

There is just a 16 k version of this, unfortunately.

**Comment by king-dahmanus** [comment]
Oh I see. Why do I feal like this voice im particular is made with
clustergen instead of hts? Or is that what hts does to lower quality audios?

**Comment by zstanecic** [comment]
Btw, to build the voice, I needed 4 hours for training.

**Comment by king-dahmanus** [comment]
so not a lot of effort for an existing text corpus along with an existing
dataset and existing language files. The trickier part is using a different
data corpus to record larger prompt selections to build possibly
higer quality voices, and the trickiest is adding a very, very very complex
language like arabic to the mix

**Comment by zstanecic** [comment]
Yep, and Grzegorz with me, and continuous research made the polish language, i.e the first version for six months.

**Comment by king-dahmanus** [comment]
wow that must be a lot. Maybe it'll take like 2 years or more to add arabic

**Comment by king-dahmanus** [comment]
also, I thought that this rhvoice had the same feature as festvox where you
could make the voice sound as high quality as 22k, even if you have an only
sixteen k dataset. Guess I shouldn't do these  comparisons a lot lolrofl

**Comment by zstanecic** [comment]
No, this HTS does to the lower quality audio.

**Comment by king-dahmanus** [comment]
wait, what do you mean? I didn't completely understand your statement
